modules/administration/pages/kubernetes.adoc

```diff
@@ -1,175 +1,173 @@
-* {productname} configured for container management
-(required channels are present, a registered container build host available, etc.)
-* [package]#virtual-host-gatherer-Kubernetes# package installed on your {productname} server
+You will also require:
 
-
-
-[[at.requirements]]
-== Requirements
-
-
-* Kubernetes version 1.5.0 or higher.
-Alternatively, you can use SUSE CaaS Platform
+* At least one {k8s} or {caasp} cluster available on your network.
+* The [systemitem]``virtual-host-gatherer-Kubernetes`` package installed on the {productname} Server.
+* {k8s} version 1.5.0 or higher, or {caasp}.
 * Docker version 1.12 or higher on the container build host
 
 
 
-[[at.registering.kubernetes.as.a.virtual.host.manager]]
-== Register Kubernetes as a Virtual Host Manager
+== Create a {k8s} Virtual Host Manager
 
+{k8s} clusters are registered with {productname} as a virtual host manager (VHM).
 
-_Kubernetes_ clusters are registered with SUSE Manager as ``virtual host managers``.
-Registration and authorization begins with importing a `kubeconfig` file using Kubernetes official command line tool ``kubectl``.
+You register and authorize your {k8s} cluster using a [path]``kubeconfig`` file with the {k8s} command line tool ``kubectl``.
 
-.Procedure: Registering a Kubernetes Cluster with {productname}
+.Procedure: Creating a {k8s} VHM
 . In the {productname} {webui}, navigate to menu:Systems[Virtual Host Managers].
-. In the [guimenu]``Create`` menu, select [guimenu]``Kubernetes Cluster``.
-. Enter a label for the new virtual host manager.
-. Select the `kubeconfig` file that contains the required data for the Kubernetes cluster.
-. Select the correct _context_ for the cluster, as specified in the kubeconfig file.
+. Click btn:[Create] and select [guimenu]``Kubernetes Cluster``.
+. In the [guimenu]``Add a Kubernetes Virtual Host Manager`` section, use these parameters:
+* In the [guimenu]``Label`` field, type a custom name for your VHM.
+* Select the [path]``kubeconfig`` file that contains the required data for the {k8s} cluster.
+. In the [guimenu]``context`` field, select the appropriate context for the cluister.
+This is specified in the [path]``kubeconfig`` file.
 . Click btn:[Create].
 
 
 
-[[at.view.the.list.of.nodes.in.a.cluster]]
-== View the List of Nodes in a Cluster
-
-. Select menu:Systems[Virtual Host Managers] from the navigation menu.
-. Select the Kubernetes cluster to view it.
-. Node data is not refreshed during registration.
-To refresh node data, click btn:[Schedule refresh data].
-. Refresh the browser.
-If the node data is not available wait a few moments, and try again.
+.Procedure: Viewing the Nodes in a Cluster
+. In the {productname} {webui}, navigate to menu:Systems[Virtual Host Managers].
+. Select the {k8s} cluster.
+. Refresh the node data by clicking btn:[Schedule refresh data].
 
+The node data can take a few moments to update.
+You might need to refresh your browser window to see the updated information.
```

Comment:
Any connection/authentication problems at this point are logged in `gatherer.log`. This might worth mentioning.

Reply:
Oops, missed this one in the last round.